website/src/components/subscriptions/notificationChannels/NotificationChannelDisplay.tsx

```diff
@@ -0,0 +1,263 @@
+import { useRef } from 'react';
+
+export type SlackChannel = {
+    name: string;
+    hook: string;
+};
+
+export type EmailChannel = {
+    address: string;
+    name: string;
+};
+
+export interface NotificationChannels {
+    email?: EmailChannel[];
+    slack?: SlackChannel[];
+}
+
+export function NotificationChannelDisplay({ notificationChannels }: { notificationChannels: NotificationChannels }) {
+    return (
+        <div className='container mx-auto min-w-72'>
+            <div className='mt-4 flex h-12 w-full place-content-between items-center rounded px-2'>
+                <h2 className='text-xl font-bold'>Notification channels</h2>
+            </div>
+
+            <div className='min-w- mt-4 flex flex-col gap-4'>
+                <div className='bordered flex flex-col gap-8 rounded-xl border-2 border-gray-200 p-4'>
+                    <div className='flex place-content-between items-center'>
+                        <div className='flex items-center gap-2'>
+                            <div className='iconify size-12 mdi--email'></div>
+                            <div className='flex flex-col'>
+                                <span className='font-bold'>Email</span>
+                                <span className='text-gray-400'>Send messages to email addresses</span>
+                            </div>
+                        </div>
+                        <AddEmail />
+                    </div>
+                    <EmailChannels channels={notificationChannels.email} />
+                </div>
+
+                <div className='bordered flex flex-col gap-8 rounded-xl border-2 border-gray-200 p-4'>
+                    <div className='flex place-content-between items-center'>
+                        <div className='flex items-center gap-2'>
+                            <div className='iconify size-12 mdi--slack'></div>
+                            <div className='flex flex-col'>
+                                <span className='font-bold'>Slack</span>
+                                <span className='text-gray-400'>Send messages to slack channels</span>
+                            </div>
+                        </div>
+                        <AddSlack />
+                    </div>
+                    <SlackChannels channels={notificationChannels.slack} />
+                </div>
+            </div>
+        </div>
+    );
+}
+
+function EmailChannels({ channels }: { channels: EmailChannel[] | undefined }) {
+    if (!channels) {
+        return <></>;
+    }
+
+    return (
+        <div>
+            {channels.map((channel, index) => (
+                <>
+                    <EmailEntry key={channel.name} channel={channel} />
+                    {index < channels.length - 1 && <hr className='my-4 border-gray-200' />}
```

Comment:
```suggestion
                    {index < channels.length - 1 && <div className='divider' />}
```
But the current one also works.

Reply:
This divider is different from the daisyui divider. It has less space above and below.